index.html

```diff
@@ -1,1642 +1,222 @@
-            <td>Number</td>
-            <td>
-              <span>Horizontal distance from the upper left corner of the touched component. The upper left corner of the component acts as the coordinate origin.</span>
-            </td>
-          </tr>	
-          <tr>
-            <th scope="row"><code>localY</code></th>
-            <td>Number</td>
-            <td>
-              <span>Vertical distance from the upper left corner of the touched component. The upper left corner of the component acts as the coordinate origin.</span>
-            </td>
-          </tr>
-          <tr>
-            <th scope="row"><code>size</code></th>
-            <td>Number</td>
-            <td>
-              <span>Touch area.</span>
-            </td>
-          </tr>			  
-		</tbody>
-	  </table>
-  
-	</section>
+    <section id="conformance">
+    </section>
+  </section>
 
+  <section id="sec-basics">
+    <h2>Component Basics</h2>
+
+    <section id="sec-pages-components">
+      <h3>MiniApp Pages and Components</h3>
+      <p>
+        A <dfn data-lt="component">MiniApp Component</dfn> is a custom MiniApp element with encapsulated functionality, data, and styles that
+        can be reused in <a href="https://www.w3.org/TR/miniapp-packaging/#dfn-page" data-link-type="dfn">MiniApp Pages</a>.
+      </p>
+      <p data-cite="HTML">
+        Each [=component=] is defined by three documents: an <a href="#sec-templating">HTML resource</a>, defining the structure and static content of the component; a <a href="#sec-stylesheets">style sheet</a> with the styles and appearance configuration for rendering; and a <a href="#sec-scripting">script resource</a> to control de business logic of the component. A component may be composed of other sub-components (stored within the package) and predefined <a href="#sec-elements">built-in elements</a>, similar to [=HTML Elements=].
+      </p>
+      <p>
+        Components and subcomponents are organized hierarchically in a tree. Although the functions and styles are encapsulated, a component may communicate with its relative nodes using <a href="#sec-events">events</a>. Developers may configure the <a href="https://www.w3.org/TR/miniapp-packaging/#dfn-page" data-link-type="dfn">Pages</a> of the app using the manifest document [[MINIAPP-MANIFEST]], pointing to the specific component associated with the page.   
+      </p>
+      <figure id="img-miniapp-pages-components">
+        <img alt="Components structure of a MiniApp (pages and components)" src="https://w3c.github.io/miniapp-packaging/images/miniapp-pages-components.svg" width="600">
+        <figcaption>Structure of a MiniApp including components and subcomponents</figcaption>
+      </figure>
+    </section>
 
-    <section id="conformance">
-    </section>      
+    <section id="sec-interface">
+      <h3>MiniApp Interface</h3>
+      <p>
+      </p>
+      <div class="ednote">
+        Formal definition of the interface?

```

Comment:
Sorry, I don't know what an interface consists of here.

In my opinion, the "properties", "lifecycle events", and "methods" below are all the interfaces

Reply:
I meant, defining the interface using IDL, similarly to how the standard elements are defined: https://dom.spec.whatwg.org/#interface-element

Reply:
Thanks for your answer. It's an important part, I totally agree with that